README.md

```diff
@@ -1 +1,32 @@
-## Battleship with Java
\ No newline at end of file
+## Battleship with Java
+
+ Battleship game built with  Java 11 and  Spring Boot 2.5.6
```

Comment:
Please clean this README up a bit. You have a couple of places here that have double spaces.

Reply:
Done!